src/migrations/20250312162155-connection-count-consumption-table.js

```diff
@@ -0,0 +1,23 @@
+
+exports.up = (db, callback) => {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS frontend_consumption_count(
```

Comment:
I called the other one connection_count_consumption so that it's not tied to backend/frontend but to connection count. This way we can change our mind when we use connections.

I'd call this one request_count_consumption so that it's related to request counting and not to frontend or backend